### PR TITLE
Added samples

### PR DESCRIPTION
**Pull request by Dexyon**
Created a set-up for adding Samples to show how our new and awesome component(s) can be used. 

Personally I always find (Ex)(S)amples very useful in projects

### CODE REVIEW

File: Samples/Example.Droid/Properties/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="com.mvvmcross.androidsupport.example">
+	<uses-sdk android:minSdkVersion="16" android:targetSdkVersion="15" />
```
Comment (martijn00) [inline Samples/Example.Droid/Properties/AndroidManifest.xml]: I think we should target v21 here

File: Samples/Example.Droid/Resources/AboutResources.txt [hunk 2]
```diff
@@ -0,0 +1,44 @@
+Images, layout descriptions, binary blobs and string dictionaries can be included
```
Comment (martijn00) [inline Samples/Example.Droid/Resources/AboutResources.txt]: This file can be removed

File: Samples/Example.Droid/Resources/layout/ListItem_SimpleTemplate.axml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (martijn00) [inline Samples/Example.Droid/Resources/layout/ListItem_SimpleTemplate.axml:2]: You are missing a reference to local here.

File: Samples/Example.Droid/packages.config [hunk 4]
```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="MvvmCross" version="3.5.2-alpha1" targetFramework="MonoAndroid44" />
```
Comment (martijn00) [inline Samples/Example.Droid/packages.config]: Please update to MonoAndroid50 after retarget api lvl v21

File: Samples/Example.Droid/Assets/AboutAssets.txt [hunk 5]
```diff
@@ -0,0 +1,19 @@
+Any raw assets you want to be deployed with your application can be placed in
```
Comment (martijn00) [inline Samples/Example.Droid/Assets/AboutAssets.txt]: This file can be removed

**Comment by martijn00** [comment]
Thanks for starting on this! We really can use some good examples. Please fix the issue's i mentioned, after that i'll merge this in and add examples for the different classes in the Android support library.

**Comment by Dexyon** [comment]
Thanks for the feedback :) Updated the Pull Request

**Comment by martijn00** [comment]
Thanks! I'll work on adding some of the other components as well.

**Pull request merged**